CONTRIBUTING.md

```diff
@@ -85,35 +85,22 @@ are researching how we can do this. If you have any ideas, please file an issue!
 #### Writing and running tests against a cloud environment
 
 If you can create cloud resources, setup your authentication using either `aws`
-or `gcloud` and set the default project as the test project. Most tests will use
-this project, unfortunately
-[some tests need a flag](https://github.com/google/go-cloud/issues/128) and will
-fail telling you what it needs to be passed. You can then run `go test` and a
-new replay file is generated. This uses API quota and will create and delete
+or `gcloud` and set the default project as the test project. Most tests will
+have constants defining the resource names they use (for example, for `blob`
```

Comment:
This seems like a pretty poor way of doing things, putting configuration into our code.
Why don't we do it via environment variables?

Reply:
It's not clear to me why that would be better. Then everyone would have to synchronize their environment variables. Anyway, this is just describing how we do things today. There already an issue about improving it to use Terraform and saved Terraform output files. #300.